### PR TITLE
Form connected to db and new chairs appearing.

### PR DESCRIPTION
**Pull request by nickclifford904**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -31,23 +31,28 @@ function displayChairs($query): string
...
 
-function insertChair()
-{
-    if(isset($_POST['submit']))
+ */
+
+function insertChair(PDO $db){
+
+    if(isset($_POST['submit'])
+        && (!empty($_POST['chairname']) && !empty($_POST['designer']) && !empty($_POST['designyear']) && !empty($_POST['imagelink'])))
```
Comment (isabellecochrane) [inline functions.php]: I think mike said if there are multiple conditionals in an if statement to wack them in a function?

Comment (isabellecochrane) [reply functions.php]: but maybe ask ash ?

File: index.php [hunk 2]
```diff
@@ -12,24 +12,28 @@
...
 <?php
 
 
-
 $result = displayChairs($query);
 echo $result;
+?>
+
+  </body>
+    </html>
```
Comment (isabellecochrane) [inline index.php:39]: strange indentation

File: functions.php (hunk 1 above)
Comment (nickclifford904) [reply functions.php]: Will do.

File: index.php (hunk 2 above)
Comment (nickclifford904) [reply index.php:39]: I don't know why. I've tried to return it to the margin and it's stuck there.

File: functions.php [hunk 3]
```diff
@@ -31,23 +31,28 @@ function displayChairs($query): string
...
 
-function insertChair()
-{
-    if(isset($_POST['submit']))
+ */
+
+function insertChair(PDO $db){
+
+    if(isset($_POST['submit'])
+        && (!empty($_POST['chairname']) && ($_POST['designer']) && ($_POST['designyear']) && ($_POST['imagelink'])))
```
Comment (ashleycoles) [inline functions.php:41]: You'll need to wrap each of thee conditions in !empty(), currently, that check is only being applied to the chairname.

**Review by ashleycoles — CHANGES_REQUESTED** [review]

File: functions.php (hunk 3 above)
Comment (nickclifford904) [reply functions.php:41]: I did that at first but then I thought the !empty might be able to cover all of them in one go and save on code.

I'll revert it to the way it was.

Cheers.

**Pull request closed without merging**